### PR TITLE
Warnings2

### PR DESCRIPTION
**Pull request by codewerfer**
Warnings in stable and unused

### CODE REVIEW

File: src/gsAssembler/gsVisitorGradGrad.h [hunk 1]
```diff
@@ -38,7 +38,7 @@ class gsVisitorGradGrad : public gsVisitorMass<T> // inherit to reuse functional
  * \f[ ( \nabla u , \nabla v )_{K} \f].
  * Where \f[ v$ \f]  is the test function and \f[u \f] is trial function.
  */
-    gsVisitorGradGrad(const gsPde<T> & pde)
+    gsVisitorGradGrad(const gsPde<T> & ) // pde)
```
Comment (stefantakacs) [inline src/gsAssembler/gsVisitorGradGrad.h]: `/*pde*/`

And the same below ...

File: src/gsCore/gsMultiPatch.h [hunk 2]
```diff
@@ -214,7 +214,7 @@ class gsMultiPatch : public gsBoxTopology, public gsFunctionSet<T>
     void addPatch(const gsGeometry<T> & g);
 
     /// \brief Search for the given geometry and return its patch index.
-    int findPatchIndex( gsGeometry<T>* g ) const;
+    std::size_t findPatchIndex( gsGeometry<T>* g ) const;
```
Comment (stefantakacs) [inline src/gsCore/gsMultiPatch.h]: Remove `std::`

and the same below

File: src/gsAssembler/gsVisitorGradGrad.h (hunk 1 above)
Comment (codewerfer) [reply src/gsAssembler/gsVisitorGradGrad.h]: OK, I see.

**Comment by stefantakacs** [comment]
@codewerfer Pls rename unittests/gsMakeUnsigned.cpp to unittests/gsMakeUnsigned_test.cpp

**Pull request merged**